templates/Web/Projects/VueDefault/src/router/index.js

```diff
@@ -0,0 +1,11 @@
+import Vue from "vue";
+import VueRouter from "vue-router";
+
+Vue.use(VueRouter);
+
+// TODO Web Template Studio: Add routes for your new pages here.
+export default new VueRouter({
+  routes: [
+
```

Comment:
Left empty for composition templates. This will be dynamically filled using routes: [      ] as context.